src/annotator/highlight-clusters.tsx

```diff
@@ -11,47 +11,55 @@ import { createShadowRoot } from './util/shadow-root';
 
 export type HighlightStyle = {
   color: string;
-  decoration: string;
+  secondColor: string;
+  thirdColor: string;
 };
 
 export type HighlightStyles = Record<string, HighlightStyle>;
 export type AppliedStyles = Record<HighlightCluster, keyof HighlightStyles>;
 
 // Available styles that users can apply to highlight clusters
 export const highlightStyles: HighlightStyles = {
-  hidden: {
+  transparent: {
     color: 'transparent',
-    decoration: 'none',
+    secondColor: 'transparent',
+    thirdColor: 'transparent',
   },
-  green: {
-    color: 'var(--hypothesis-color-green)',
-    decoration: 'none',
+  pink: {
+    color: 'var(--hypothesis-color-pink)',
+    secondColor: 'var(--hypothesis-color-pink-1)',
+    thirdColor: 'var(--hypothesis-color-pink-2)',
   },
   orange: {
     color: 'var(--hypothesis-color-orange)',
-    decoration: 'none',
+    secondColor: 'var(--hypothesis-color-orange-1)',
+    thirdColor: 'var(--hypothesis-color-orange-2)',
   },
-  pink: {
-    color: 'var(--hypothesis-color-pink)',
-    decoration: 'none',
+  yellow: {
+    color: 'var(--hypothesis-color-yellow)',
+    secondColor: 'var(--hypothesis-color-yellow-1)',
+    thirdColor: 'var(--hypothesis-color-yellow-2)',
+  },
+  green: {
+    color: 'var(--hypothesis-color-green)',
+    secondColor: 'var(--hypothesis-color-green-1)',
+    thirdColor: 'var(--hypothesis-color-green-2)',
   },
   purple: {
     color: 'var(--hypothesis-color-purple)',
-    decoration: 'none',
-  },
-  yellow: {
-    color: 'var(--hypothesis-color-yellow)',
-    decoration: 'none',
+    secondColor: 'var(--hypothesis-color-purple-1)',
+    thirdColor: 'var(--hypothesis-color-purple-2)',
   },
   grey: {
     color: 'var(--hypothesis-color-grey)',
-    decoration: 'underline dotted',
+    secondColor: 'var(--hypothesis-color-grey-1)',
+    thirdColor: 'var(--hypothesis-color-grey-2)',
   },
 };
 
 // The default styles applied to each highlight cluster. For now, this is
 // hard-coded.
-export const defaultStyles: AppliedStyles = {
+export const defaultClusterStyles: AppliedStyles = {
```

Comment:
A few light refactors here:

* rename `defaultStyles` => `defaultClusterStyles`
* rename `_setClusterStyle` to `_setClusterStyles` (as it sets multiple) and new convenience `_setClusterStyle` that sets an individual CSS var